### PR TITLE
style: use tabs in dashboard edit pane

### PR DESCRIPTION
**Pull request by mistercrunch**
Currently the east pane in the "edit dashboard" flow is really confusing. The "Components" and "Colors" buttons at the top really act as tabs, and the "slider" pattern is kind of cool but confusing.

## after (draft - general direction)
Screen Shot 2020-07-23 at 11 47 55 PM
Screen Shot 2020-07-23 at 11 47 38 PM

## before
Screen Shot 2020-07-22 at 9 20 16 AM

- just noticed that Colors could be moved to properties too - maybe

### CODE REVIEW

**Comment by Steejay** [comment]
@mistercrunch wondering if we should change order to charts>components>colors? Charts are the main element of a dashboard.

**Comment by mistercrunch** [comment]
I have not usage data to back this up, but for me I always "Save and add to dashboard" from explore and almost never use the chart panel in the "dashboard editor"

**Comment by ktmud** [comment]
Never even noticed the "Colors" setting. Agreed this is weird.

I use the Charts panel a lot. Sometimes you want to relayout the dashboard and it's easier to delete a chart first then re-add it later. Wondering if we can just keep the "Insert Components" panel as is and move "Set Color Scheme" to a modal triggered by a link in the dropdown menu?

image


Or we can change "Edit CSS" to "Change Appearance" and move color scheme selector to the Edit CSS modal. The same modal may even hold a "Select Theme" control in the future.

File: superset-frontend/src/dashboard/components/AddSliceCard.jsx [hunk 1]
```diff
@@ -62,9 +66,7 @@ function AddSliceCard({
           </div>
           <div className="item">
             <span>{t('Data source')} </span>
-            <span // eslint-disable-next-line react/no-danger
-              dangerouslySetInnerHTML={{ __html: datasourceLink }}
-            />
+            <a href={datasourceUrl}>{datasourceName}</a>
```
Comment (mistercrunch) [inline superset-frontend/src/dashboard/components/AddSliceCard.jsx:69]: Bycatch - moving away from `dangerouslySetInnerHTML `

**Comment by mistercrunch** [comment]
@ktmud I did a lot of work to untangle the sliders and move towards tabs as defined in SIP-34 here, I'd hate to roll this back.
Screen Shot 2020-07-28 at 5 04 28 PM

**Comment by ktmud** [comment]
Makes sense. I can see why it'd be useful to have the tabs.


Here's another screenshot of SIP-34 design with proper tab titles and icons:

**Comment by mistercrunch** [comment]
Oh I've been using Figma and I think something went wrong in the Sketch -> Figma transition in this area ... @Steejay ^^^

**Review by suddjian — APPROVED** [review]
lgtm

**Pull request merged**